include/mitsuba/render/bsdf.h

```diff
@@ -519,6 +519,23 @@ class MI_EXPORT_LIB BSDF : public Object {
     virtual Spectrum eval_diffuse_reflectance(const SurfaceInteraction3f &si,
                                               Mask active = true) const;
 
+    /**
+     * \brief Evaluate roughness
+     *
+     * This method approximates the roughness for a given
+     * direction. For some materials, an exact value can be computed
+     * inexpensively.
+     * When this is not possible, the value is approximated by
+     * evaluating the BSDF for a normal outgoing direction and returning this
+     * value multiplied by pi. This is the default behaviour of this method.
+     *
+     * \param si
+     *     A surface interaction data structure describing the underlying
+     *     surface position.
+     */
+    virtual Float eval_roughness(const SurfaceInteraction3f &si,
+                                              Mask active = true) const;
```

Comment:
Indentation?

Reply:
Do you mean I should indent it one more tab or one less?
Isn't it currently in line with `diffuse_reflectance`?

Reply:
The `Mask active = true` argument should be aligned with the opening parenthesis of the previous line. Like in the other methods. Do you see what I mean?